docker/planetscale_proxy/Dockerfile

```diff
@@ -9,7 +9,7 @@ ENTRYPOINT /go/bin/ps-http-sim \
   -http-port=8085 \
   -mysql-addr=$MYSQL_HOST \
   -mysql-port=$MYSQL_PORT \
-  -mysql-idle-timeout=5s \
+  -mysql-idle-timeout=1200s \
```

Comment:
Aims to fix the following error, keep an eye out if you see this again and please let me know if you do. The source code of this planetscale proxy says that the timeout code has race conditions, so I aimed at a very high timeout to see if we can fix this.

```
 DatabaseError: Code: INTERNAL
    invalid sequence, expected X got Y
     (errno 2013) (sqlstate HY000) during query: SELECT `test-0000-00000000`.`User`.`id` FROM `test-0000-00000000`.`User` WHERE (`test-0000-00000000`.`User`.`id` = 'a1e8d6df4d7adb11db8b0ac9' AND 1=1) LIMIT 1 OFFSET 0

      at Connection.execute (../../../../node_modules/.pnpm/@planetscale+database@1.11.0/node_modules/@planetscale/database/dist/cjs/index.js:99:19)
      at PrismaPlanetScale.performIO (../../../../node_modules/.pnpm/@prisma+adapter-planetscale@0.4.0_@planetscale+database@1.11.0/node_modules/@prisma/adapter-planetscale/dist/index.js:143:22)
      at PrismaPlanetScale.queryRaw (../../../../node_modules/.pnpm/@prisma+adapter-planetscale@0.4.0_@planetscale+database@1.11.0/node_modules/@prisma/adapter-planetscale/dist/index.js:114:54)
      at ../../../../node_modules/.pnpm/@prisma+driver-adapter-utils@0.8.0/node_modules/@prisma/driver-adapter-utils/dist/index.mjs:71:14
```